env/JET.env

```diff
@@ -103,12 +104,8 @@ elif [ $step = "fcst" ]; then
     export NTHREADS_FV3=${nth_fv3:-$nth_max}
     [[ $NTHREADS_FV3 -gt $nth_max ]] && export NTHREADS_FV3=$nth_max
     export cores_per_node=$npe_node_max
-    #export APRUN_FV3="$launcher ${npe_fv3:-${npe_fcst:-$PBS_NP}}"
-    if [ $CDUMP = "gdas" ]; then
-      export APRUN_FV3="$launcher ${npe_fcst:-$PBS_NP}"
-    else
-      export APRUN_FV3="$launcher ${npe_fcst_gfs:-$PBS_NP}"
-    fi
+    export APRUN_FV3="$launcher"
```

Comment:
Possibly, I'd need to see the issue in the sprint branch more to understand...but this works on Hera and Jet without specifying a task number. Adding a npe value behind the $launcher setting (as it was in JET.env) causes an error. The contents of JET.env were very outdated so this brings APRUN_FV3 in line with changes in the other env files on SLURM machines.

Reply:
On Orion, we were over-requesting CPU resources relative to the layout (240 tasks vs 224 needed) and FV3 was getting the 240 from srun and complaining. This might be a recent change to UFS that will have to be addressed everywhere if we are over-requesting elsewhere.

Reply:
Gotcha, we can address that in a separate issue.